workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/MembersPage/MembersPage.tsx

```diff
@@ -235,8 +238,10 @@ export function MembersPage({
 				>
 					{selectedMember ? (
 						<MemberProfile
-							member={selectedMember}
+							memberUser={selectedMember}
+							renderToast={renderToast}
 							setSelectedMember={setSelectedMember}
+							userLogged={userLogged}
 						></MemberProfile>
```

Comment:
You can close directly the component
```suggestion
						/>
```